.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+known_local_folder=app
```

Comment:
isort did not recognize the app folder hence was sorting it with 3rd party packages (not my own) - https://pybit.es/articles/debugging-tale/